<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimWindow</class>
 <widget class="QWidget" name="SimWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot window</string>
  </property>
  <widget class="QCustomPlot" name="customPlot_CVs" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>641</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CancelSim">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SaveSim">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_autoFill">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Autofill -&gt; Potential Program</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
